package.json

```diff
@@ -28,7 +28,8 @@
     "lint:js": "eslint .",
     "start": "ember serve",
     "start:dev": "ember server --proxy https://conduit.productionready.io",
-    "test": "ember test"
+    "test": "npm-run-all lint test:*",
```

Comment:
I noticed that previously, the line showed `lint:*`. I created a 3.24 app and the line showed `lint` only. I believe either will work.